js/script.js

```diff
@@ -113,39 +117,76 @@ const createTaskCard = (task) => {
+      <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M20.285 2L9 13.567L3.714 8.556L0 12.272L9 21L24 5.715L20.285 2Z" fill="#BBBDD0"/>
+      </svg>
+    `;
+    checkButton.addEventListener("click", () => {
+      task.isCompleted = !task.isCompleted;
+      taskCard.classList.toggle("task-completed");
+      renderTasks(currentFilter);
+    });
+    buttonsContainer.appendChild(checkButton);
+
     const editButton = document.createElement("button");
     editButton.className = "btn-edit";
-    editButton.textContent = "Edit";
+    editButton.innerHTML = `
+      <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <g clip-path="url(#clip0_3_35)">
+          <path d="M7.127 22.562L0 24L1.438 16.872L7.127 22.562ZM8.541 21.148L19.769 9.923L14.079 4.231L2.852 15.458L8.541 21.148ZM18.309 0L15.493 2.817L21.184 8.508L24 5.689L18.309 0V0Z" fill="#BBBDD0"/>
+        </g>
+        <defs>
+          <clipPath id="clip0_3_35">
+            <rect width="24" height="24" fill="white"/>
+          </clipPath>
+        </defs>
+      </svg>
+    `;
     editButton.addEventListener("click", () => {
-      editTask(task.id);
+      task.isBeingEdited = true;
+      renderTasks(currentFilter);
     });
-    taskCard.appendChild(editButton);
+    buttonsContainer.appendChild(editButton);
 
     const deleteButton = document.createElement("button");
     deleteButton.className = "btn-delete";
-    deleteButton.textContent = "Delete";
+    deleteButton.innerHTML = `
+      <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path fill-rule="evenodd" clip-rule="evenodd" d="M19 24H5C3.896 24 3 23.104 3 22V6H21V22C21 23.104 20.104 24 19 24ZM10 10C10 9.448 9.552 9 9 9C8.448 9 8 9.448 8 10V19C8 19.552 8.448 20 9 20C9.552 20 10 19.552 10 19V10ZM16 10C16 9.448 15.552 9 15 9C14.448 9 14 9.448 14 10V19C14 19.552 14.448 20 15 20C15.552 20 16 19.552 16 19V10ZM22 5H2V3H8V1.5C8 0.673 8.673 0 9.5 0H14.5C15.325 0 16 0.671 16 1.5V3H22V5ZM10 3H14V2H10V3Z" fill="#BBBDD0"/>
+      </svg>
+    `;
     deleteButton.addEventListener("click", () => {
       deleteTask(task.id);
     });
-    taskCard.appendChild(deleteButton);
-  }
+    buttonsContainer.appendChild(deleteButton);
 
-  if (!task.isCompleted) {
-    const checkButton = document.createElement("button");
-    checkButton.className = "btn-check";
-    checkButton.textContent = "✔️";
-    checkButton.addEventListener("click", () => {
-      task.isCompleted = true;
-      renderTasks();
-    });
-    taskCard.appendChild(checkButton);
+    actionsContainer.appendChild(buttonsContainer);
+
+    if (task.isCompleted) {
+      const completedTag = document.createElement("div");
+      completedTag.className = "completed-tag";
+      completedTag.textContent = "Completed in 3 days";
+      actionsContainer.appendChild(completedTag);
```

Comment:
make a function in `utility.js` for counting completion time of a task and use it to show individual task completion time. Don't use `hard coded text`.